core/server/master/src/main/java/alluxio/master/file/meta/LockedInodePath.java

```diff
@@ -159,6 +158,9 @@ public LockedInodePath(AlluxioURI uri, ReadOnlyInodeStore inodeStore,
     mLockList = new SimpleInodeLockList(inodeLockManager, mUseTryLock);
     mTracker = DETECTOR.track(this);
     mJournalContext = journalContext;
+    mMergeInodeJournals = Configuration.getBoolean(
+        PropertyKey.MASTER_FILE_SYSTEM_MERGE_INODE_JOURNALS
+    ) && mJournalContext instanceof FileSystemMergeJournalContext;
```

Comment:
is it possible that MASTER_FILE_SYSTEM_MERGE_INODE_JOURNALS changes at runtime? Otherwise I think it's still possible to resolve the property statically.

Reply:
it cannot but in the unit tests we want test with different property values. Making it a member field so that we are able to change the configurations and create different class instances with mMergeInodeJournals values